### ISSUE TITLE
Ramps vs Mainline.   The data collection method is not user friendly for locates with multiple linear reference segments in close proximity like interchanges at junctions of state routes.  If there is not a tool or field collection method that allows the user to "tie" the feature to the desired linear reference, then maybe the mainline lrs and ramp lrs  should be loaded separately into HATS so the user can toggle off ramps or vice versa so the feature can be properly located?

### ISSUE THREAD

**Issue by HATSBot** [WSDOT/HATS#622]
- **Device Name:** ERD1181001
- **Device Model:** iPad Air
- **Device System Version:** 10.0.2
- **Application Version:** HATS-iOS 2.1.1
- **Name:** M Melvin
- **Has Attachments:** False
- **Feedback:** Ramps vs Mainline.   The data collection method is not user friendly for locates with multiple linear reference segments in close proximity like interchanges at junctions of state routes.  If there is not a tool or field collection method that allows the user to &quot;tie&quot; the feature to the desired linear reference, then maybe the mainline lrs and ramp lrs  should be loaded separately into HATS so the user can toggle off ramps or vice versa so the feature can be properly located?
- **Details:**  <a href='http://hatsprod.wsdot.loc/Maintenance/management/activities/Feedback/GetMobileDeviceFeedback?mobileFeedbackId=1977'>View Details</a>

**Comment by joeschmit** [comment]
Hi Mike –

So to cut to the chase on this, we aren’t going to be changing the method in which we derive SRMP data for our inventoried HATS features.  Currently, the process of collecting geographic points meets our business requirements here in the maintenance program.  We have tried other methods as you have described in the past with very limited success.  That being said, we definitely want to ensure we are collecting quality data.  We feel that the data is most accurate and provides the most value to us/the agency, when geographic points are collect versus referencing the LRS (not that accurate).  If features are not collected with proper technique and diligence the data can obviously be misconstrued.  That is why we have spent the time to create “location update” tools in the HATS mobile app, that allows the location of any feature to be changed to achieve a higher degree of accuracy.

I hope this helps shed some more light on why things are the way they are.  Thank you for sending us your feedback.

Cheers –

Joe Schmit
